contrib/kind.sh

```diff
@@ -20,13 +20,16 @@ run_kubectl() {
   done
 }
 
+MNODE_COUNT=1
 K8S_VERSION=${K8S_VERSION:-v1.17.2}
 KIND_INSTALL_INGRESS=${KIND_INSTALL_INGRESS:-false}
 KIND_HA=${KIND_HA:-false}
 if [ "$KIND_HA" == true ]; then
   DEFAULT_KIND_CONFIG=./kind-ha.yaml
+  MNODE_COUNT=$(eval "grep \"^\s-\srole\s*:\s*control-plane\" kind-ha.yaml | wc -l")
```

Comment:
The `eval` here (and below) is unnecessary. Also, you can use `grep -c` rather than piping to `wc -l`

Reply:
thanks for the suggestion, shall modify to 
```
MNODE_COUNT=`grep -c "^\s-\srole\s*:\s*control-plane" kind-ha.yaml`
MNODE_COUNT=`grep -c "^\s-\srole\s*:\s*control-plane" kind.yaml`
```